lib/ui/painting.dart

```diff
@@ -2215,7 +2214,7 @@ class Path extends NativeFieldWrapperClass2 {
   /// The resulting path will be constructed from non-overlapping contours. The
   /// curve order is reduced where possible so that cubics may be turned into
   /// quadratics, and quadratics maybe turned into lines.
-  static Path combine(PathOperation operation, Path path1, Path path2) {
+  static Path/*!*/ combine(PathOperation/*!*/ operation, Path/*!*/ path1, Path/*!*/ path2) {
     assert(path1 != null);
     assert(path2 != null);
     final Path path = Path();
```

Comment:
Does `_op` down on 2226 need to be annotated?

Reply:
I currently don't spend too much time on annotating internals, because they can be cleaned up later without disruption. Also the migration tool is pretty good at deducing the correct nullability annotations for internal parts based on the public API.

Reply:
Having said that. Done.